packages/react-ui/src/features/flows/components/select-flow-template-dialog.tsx

```diff
@@ -0,0 +1,122 @@
+import { useMutation } from '@tanstack/react-query';
```

Comment:
Let's discuss the structure of the project (I am kind of hesitant now )

I suggest moveing this the app level, since it doesn't use features template and pieces.